my_cv/css/style.css

```diff
@@ -0,0 +1,92 @@
+*{
+    text-decoration: none;
+    list-style:circle;
+
+}
+
+body {
+    max-width: 1024px;
+    margin: auto;
+    padding: 5px;
+    display: block;
```

Comment:
here you set the display property last, but in all other examples it came first. It's a good habit to have the same order of rules based on their importance - https://9elements.com/css-rule-order/